alacritty_terminal/src/term/mod.rs

```diff
@@ -756,12 +756,17 @@ impl Damage {
     #[inline(always)]
     fn resize(&mut self, lines: Line, num_cols: Column) {
         self.num_cols = num_cols;
-        self.line_damage = Vec::with_capacity(lines.0);
+        self.lines = Vec::with_capacity(lines.0);
         for line in 0..lines.0 {
-            self.line_damage.push(LineDamage::undamaged(Line(line), num_cols));
+            self.lines.push(LineDamage::undamaged(Line(line), num_cols));
         }
         self.damage_all = true;
     }
+
+    #[allow(dead_code)]
+    fn damaged_lines(&self) -> usize {
```

Comment:
You could add one more `impl` in your test cfg handler iirc, right? So this code will be in a right place, I guess. Anyway having a `dead_code` here is a bit odd, so I feel like adding a free function in test mod should be a good idea.

Reply:
Ah, much prettier. Wasn't sure if I could do that. :)